docs/src/pages/ui/primitives/tabs/demo.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { Tabs, TabItem, Flex, Button, Text } from '@aws-amplify/ui-react';
+import { TabsPropControls } from '@/components/TabsPropControls';
+import { useTabsProps } from '@/components/useTabsProps';
+import { Example } from '@/components/Example';
+
+const demoChildren = [
+  <TabItem title="Greeting">
+    <Button>Hello</Button>
+  </TabItem>,
+  <TabItem title="Foo">Bar</TabItem>,
+  <TabItem title="Disabled for demonstration" isDisabled={true}>
+    Cannot click
+  </TabItem>,
+];
+
+export const TabsDemo = () => {
+  const tabsProps = useTabsProps({
+    defaultTab: 0,
+    grow: '',
+    justifyContent: 'flex-start',
+    children: demoChildren.map((child) => child),
```

Comment:
This was done to avoid a TypeScript error. Perhaps there's a more elegant way to set the children for the demo.

Reply:
I wasn't getting this Typescript error, removed this map and everything works!